projects/azure/README.md

```diff
@@ -0,0 +1,17 @@
+# Test Ci-operator jobs 
```

Comment:
Need to add a more general description of the azure projects here and then on a section talk about the ci-operator.